torch/csrc/utils/structseq.cpp

```diff
@@ -0,0 +1,102 @@
+/* Copyright Python Software Foundation
+ *
+ * This file is copy-pasted from CPython source code with modifications:
+ * https://github.com/python/cpython/blob/master/Objects/structseq.c
+ * https://github.com/python/cpython/blob/2.7/Objects/structseq.c
+ *
+ * The purpose of this file is to overwrite the default behavior
+ * of repr of structseq to provide better printting for returned
+ * structseq objects from operators, aka torch.return_types.*
+ *
+ * For more information on copyright of CPython, see also:
+ * https://github.com/python/cpython#copyright-and-license-information
+ */
+
+#include "torch/csrc/utils/structseq.h"
+#include "torch/csrc/utils/six.h"
+#include "structmember.h"
+#include <sstream>
+
+namespace torch {
+namespace utils {
+
+#if PY_MAJOR_VERSION == 2
+PyObject *structseq_slice(PyStructSequence *obj, Py_ssize_t low, Py_ssize_t high)
+{
+    PyTupleObject *np;
+    Py_ssize_t i;
+
+    if (low < 0)
```

Comment:
nit:  do you mind to enclose all if/for/while/do-while blocks in curly braces. This will surely increase maintainability of the code.

Reply:
@VitalyFedyunin Sure, done!